src/events/cd-event-add-child-ticket.vue

```diff
@@ -0,0 +1,182 @@
+       <h3 class="cd-child-ticket__ticket-name">{{ $t('Ticket')}} {{firstName|addPosession}} </h3>
+    </div>
+    <div class="cd-child-ticket__body">
+      <label>{{ $t('Name')}}</label>
+      <div class="cd-child-ticket__child-name">
+        <input class="cd-child-ticket__first-name form-control" v-model="firstName" type="firstName" :placeholder="$t('First Name')" data-vv-name="firstName" data-vv-validate-on="blur" v-validate="'required'"/>
+        <input class="cd-child-ticket__surname form-control" v-model="surname" type="surname" :placeholder="$t('Surname')" data-vv-name="surname" data-vv-validate-on="blur" v-validate="'required'"/>
+      </div>
+      <p class="cd-child-ticket__first-name-err text-danger" v-show="errors.has('firstName:required')">{{ $t('First name is required') }}</p>
+      <p class="cd-child-ticket__surname-err text-danger" v-show="errors.has('surname:required')">{{ $t('Surname is required') }}</p>
+
+      <label>{{ $t('Date of Birth') }}</label>
+      <div class="cd-child-ticket__dob-picker-wrapper">
+        <vue-dob-picker select-class="form-control" v-model="dob" 
+                        show-labels="false" month-format="short"
+                        :placeholders="[$t('Date'), $t('Month'), $t('Year')]"
+                        :proportions="[2, 2, 3]" data-vv-value-path="value" data-vv-name="dob" v-validate="'required'"></vue-dob-picker>                       
+      </div> 
+      <p class="cd-child-ticket__dob-err text-danger" v-show="errors.has('dob:required')">{{ $t('Date of Birth is required') }}</p>      
+
+      <label>{{ $t('Gender') }}</label>
+      <div class="cd-child-ticket__gender-selector">
+        <select class="cd-child-ticket__select-box form-control" data-vv-name="gender" data-vv-validate-on="blur" v-validate="'required'" v-model="genderSelect">
+          <option value="" selected data-default disabled>{{ $t('Select an option') }}</option>
+          <option value="male">{{ $t('Male') }}</option>
+          <option value="female">{{ $t('Female') }}</option>
+          <option value="prefer not to answer">{{ $t('Prefer not to answer') }}</option>
+          <option value="specify">{{ $t('Specify Identity') }}</option>
+        </select>
+        <input v-show="specifyGender" class="form-control" v-model="genderInput" :placeholder="$t('Identify as...')" data-vv-name="gender" data-vv-validate-on="blur" v-validate="specifyGender ? 'required' : ''"/>
+      </div>
+      <p class="cd-child-ticket__gender-err text-danger" v-show="errors.has('gender:required')">{{ $t('Gender is required') }} <a v-on:click="showWhy">{{ $t('Why is this required? Click Here To Find Out More') }}</a><p v-show="whyGender && errors.has('gender:required')">{{ $t(`We want to provide activities that appeal to people regardless of their gender. To check how well we are succeeding, we'd like to find out whether or not people of different genders are equally likely to take part.`) }}</p></p> 
```

Comment:
Jump line here please :p This is way too long and I'm surprised we don't have a linter crying over that length

Reply:
Also a <p> inside a <p> is invalid, use a span inside rather

Reply:
I just changed it to another paragraph element outside that error instead of using the span. If i use the span it inherits the text-danger class which is not nice